### ISSUE TITLE
v1.6 docs not building on readthedocs

### ISSUE THREAD

**Issue by ThomasAkam** [pyControl/docs#14]
I'm trying to make some old versions of the docs available on readthedocs.  The v1.7 docs build fine but v1.6 docs gave an error.  I've created a v1.6 branch and tried to update them with a new `readthedocs.yml` and `requirements.txt` taken from the newer versions that build OK, but they are still giving an error when I try and build them on readthedocs (reproduced below).  It looks like readthedocs is trying to build them using Python 3.7 even though python 3.9 is specified in `readthedocs.yml`. @alustig3 do you have any idea what the problem might be?

```
WARNING -  Config value: 'nav'. Warning: Unrecognised configuration name: nav 
INFO    -  Cleaning site directory 
INFO    -  Building documentation to directory: /home/docs/checkouts/readthedocs.org/user_builds/pycontrol/checkouts/v1.6/_build/html 
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/pycontrol/envs/v1.6/lib/python3.7/site-packages/mkdocs/nav.py", line 337, in _follow
    raise StopIteration
StopIteration

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/docs/.pyenv/versions/3.7.9/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/docs/.pyenv/versions/3.7.9/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/docs/checkouts/readthedocs.org/user_builds/pycontrol/envs/v1.6/lib/python3.7/site-packages/mkdocs/__main__.py", line 194, in <module>
    cli()
  File "/home/docs/checkouts/readthedocs.org/user_builds/pycontrol/envs/v1.6/lib/python3.7/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/docs/checkouts/readthedocs.org/user_builds/pycontrol/envs/v1.6/lib/python3.7/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/docs/checkouts/readthedocs.org/user_builds/pycontrol/envs/v1.6/lib/python3.7/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/docs/checkouts/readthedocs.org/user_builds/pycontrol/envs/v1.6/lib/python3.7/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/docs/checkouts/readthedocs.org/user_builds/pycontrol/envs/v1.6/lib/python3.7/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/docs/checkouts/readthedocs.org/user_builds/pycontrol/envs/v1.6/lib/python3.7/site-packages/mkdocs/__main__.py", line 156, in build_command
    ), dirty=not clean)
  File "/home/docs/checkouts/readthedocs.org/user_builds/pycontrol/envs/v1.6/lib/python3.7/site-packages/mkdocs/commands/build.py", line 282, in build
    build_pages(config, dirty=dirty)
  File "/home/docs/checkouts/readthedocs.org/user_builds/pycontrol/envs/v1.6/lib/python3.7/site-packages/mkdocs/commands/build.py", line 213, in build_pages
    site_navigation = nav.SiteNavigation(config)
  File "/home/docs/checkouts/readthedocs.org/user_builds/pycontrol/envs/v1.6/lib/python3.7/site-packages/mkdocs/nav.py", line 44, in __init__
    config, self.url_context)
  File "/home/docs/checkouts/readthedocs.org/user_builds/pycontrol/envs/v1.6/lib/python3.7/site-packages/mkdocs/nav.py", line 392, in _generate_site_navigation
    config_line, url_context, config):
RuntimeError: generator raised StopIteration
```

**Comment by alustig3** [comment]
hmm...

I was able to get both the latest and v1.6 built: https://andy-pycontrol-docs.readthedocs.io/en/latest/ and https://andy-pycontrol-docs.readthedocs.io/en/v1.6/

Perhaps you have an environmental variable overriding things? I didn't have any Environment Variables setup: 
![CleanShot 2022-10-11 at 10 41 42](https://user-images.githubusercontent.com/8128628/195151179-20fa68e4-dc54-4d50-92f5-0c3ea775ab68.png)

It's weird that your build is using Python 3.7 instead of what is specified in the `readthedocs.yml`, but even so, I think it should still build using 3.7? I first built with without any changes to the v1.6 branch and it used Python 3.9. I then changed to 3.10 in the `readthedocs.yml` to confirm that the `readthedocs.yml` file was being used and I didn't have some other variable specifying python 3.9. Finally, I changed the `readthedocs.yml` to specify 3.7 and docs were still successfully built. Below is the raw output from my v1.6 build using Python 3.7. Maybe you can compare it to your output?


```
Read the Docs build information
Build id: 18311022
Project: andy-pycontrol-docs
Version: v1.6
Commit: c5965ca161a20ad5287aaad21ddf3f2af0d402a4
Date: 2022-10-11T16:55:15.587322Z
State: finished
Success: True


[rtd-command-info] start-time: 2022-10-11T16:55:17.926816Z, end-time: 2022-10-11T16:55:18.671943Z, duration: 0, exit-code: 0
git clone --no-single-branch --depth 50 https://github.com/alustig3/docs.git .
Cloning into '.'...

[rtd-command-info] start-time: 2022-10-11T16:55:18.961884Z, end-time: 2022-10-11T16:55:19.063407Z, duration: 0, exit-code: 0
git checkout --force origin/v1.6
Note: switching to 'origin/v1.6'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at c5965ca error if 3.7?

[rtd-command-info] start-time: 2022-10-11T16:55:19.380014Z, end-time: 2022-10-11T16:55:19.444047Z, duration: 0, exit-code: 0
git clean -d -f -f


[rtd-command-info] start-time: 2022-10-11T16:55:24.500899Z, end-time: 2022-10-11T16:55:24.581917Z, duration: 0, exit-code: 0
asdf global python 3.7.13


[rtd-command-info] start-time: 2022-10-11T16:55:25.145562Z, end-time: 2022-10-11T16:55:26.247094Z, duration: 1, exit-code: 0
python -mvirtualenv /home/docs/checkouts/readthedocs.org/user_builds/andy-pycontrol-docs/envs/v1.6
created virtual environment CPython3.7.13.final.0-64 in 756ms
  creator CPython3Posix(dest=/home/docs/checkouts/readthedocs.org/user_builds/andy-pycontrol-docs/envs/v1.6, clear=False, no_vcs_ignore=False, global=False)
  seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/docs/.local/share/virtualenv)
    added seed packages: pip==21.2.3, setuptools==57.4.0, wheel==0.37.0
  activators BashActivator,CShellActivator,FishActivator,PowerShellActivator,PythonActivator

[rtd-command-info] start-time: 2022-10-11T16:55:26.437589Z, end-time: 2022-10-11T16:55:34.622964Z, duration: 8, exit-code: 0
/home/docs/checkouts/readthedocs.org/user_builds/andy-pycontrol-docs/envs/v1.6/bin/python -m pip install --upgrade --no-cache-dir pip setuptools<58.3.0
Requirement already satisfied: pip in /home/docs/checkouts/readthedocs.org/user_builds/andy-pycontrol-docs/envs/v1.6/lib/python3.7/site-packages (21.2.3)
Collecting pip
  Downloading pip-22.2.2-py3-none-any.whl (2.0 MB)
Requirement already satisfied: setuptools<58.3.0 in /home/docs/checkouts/readthedocs.org/user_builds/andy-pycontrol-docs/envs/v1.6/lib/python3.7/site-packages (57.4.0)
Collecting setuptools<58.3.0
  Downloading setuptools-58.2.0-py3-none-any.whl (946 kB)
Installing collected packages: setuptools, pip
  Attempting uninstall: setuptools
    Found existing installation: setuptools 57.4.0
    Uninstalling setuptools-57.4.0:
      Successfully uninstalled setuptools-57.4.0
  Attempting uninstall: pip
    Found existing installation: pip 21.2.3
    Uninstalling pip-21.2.3:
      Successfully uninstalled pip-21.2.3
Successfully installed pip-22.2.2 setuptools-58.2.0

[rtd-command-info] start-time: 2022-10-11T16:55:34.809462Z, end-time: 2022-10-11T16:55:45.002684Z, duration: 10, exit-code: 0
/home/docs/checkouts/readthedocs.org/user_builds/andy-pycontrol-docs/envs/v1.6/bin/python -m pip install --upgrade --no-cache-dir pillow mock==1.0.1 alabaster>=0.7,<0.8,!=0.7.5 commonmark==0.9.1 recommonmark==0.5.0 mkdocs<1.1
Collecting pillow
  Downloading Pillow-9.2.0-cp37-cp37m-manylinux_2_28_x86_64.whl (3.2 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.2/3.2 MB 44.3 MB/s eta 0:00:00
Collecting mock==1.0.1
  Downloading mock-1.0.1.zip (861 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 861.9/861.9 kB 138.1 MB/s eta 0:00:00
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Collecting alabaster!=0.7.5,<0.8,>=0.7
  Downloading alabaster-0.7.12-py2.py3-none-any.whl (14 kB)
Collecting commonmark==0.9.1
  Downloading commonmark-0.9.1-py2.py3-none-any.whl (51 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 51.1/51.1 kB 99.8 MB/s eta 0:00:00
Collecting recommonmark==0.5.0
  Downloading recommonmark-0.5.0-py2.py3-none-any.whl (9.8 kB)
Collecting mkdocs<1.1
  Downloading mkdocs-1.0.4-py2.py3-none-any.whl (1.2 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.2/1.2 MB 205.7 MB/s eta 0:00:00
Collecting docutils>=0.11
  Downloading docutils-0.19-py3-none-any.whl (570 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 570.5/570.5 kB 122.7 MB/s eta 0:00:00
Collecting sphinx>=1.3.1
  Downloading sphinx-5.2.3-py3-none-any.whl (3.2 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.2/3.2 MB 143.2 MB/s eta 0:00:00
Collecting click>=3.3
  Downloading click-8.1.3-py3-none-any.whl (96 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 96.6/96.6 kB 193.9 MB/s eta 0:00:00
Collecting Markdown>=2.3.1
  Downloading Markdown-3.4.1-py3-none-any.whl (93 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 93.3/93.3 kB 180.8 MB/s eta 0:00:00
Collecting tornado>=5.0
  Downloading tornado-6.2-cp37-abi3-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (423 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 424.0/424.0 kB 213.6 MB/s eta 0:00:00
Collecting Jinja2>=2.7.1
  Downloading Jinja2-3.1.2-py3-none-any.whl (133 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 133.1/133.1 kB 200.8 MB/s eta 0:00:00
Collecting livereload>=2.5.1
  Downloading livereload-2.6.3.tar.gz (25 kB)
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Collecting PyYAML>=3.10
  Downloading PyYAML-6.0-cp37-cp37m-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_12_x86_64.manylinux2010_x86_64.whl (596 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 596.3/596.3 kB 140.6 MB/s eta 0:00:00
Collecting importlib-metadata
  Downloading importlib_metadata-5.0.0-py3-none-any.whl (21 kB)
Collecting MarkupSafe>=2.0
  Downloading MarkupSafe-2.1.1-cp37-cp37m-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (25 kB)
Collecting six
  Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)
Collecting sphinxcontrib-htmlhelp>=2.0.0
  Downloading sphinxcontrib_htmlhelp-2.0.0-py2.py3-none-any.whl (100 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100.5/100.5 kB 197.8 MB/s eta 0:00:00
Collecting snowballstemmer>=2.0
  Downloading snowballstemmer-2.2.0-py2.py3-none-any.whl (93 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 93.0/93.0 kB 194.1 MB/s eta 0:00:00
Collecting babel>=2.9
  Downloading Babel-2.10.3-py3-none-any.whl (9.5 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 9.5/9.5 MB 136.0 MB/s eta 0:00:00
Collecting imagesize>=1.3
  Downloading imagesize-1.4.1-py2.py3-none-any.whl (8.8 kB)
Collecting requests>=2.5.0
  Downloading requests-2.28.1-py3-none-any.whl (62 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 62.8/62.8 kB 151.0 MB/s eta 0:00:00
Collecting packaging>=21.0
  Downloading packaging-21.3-py3-none-any.whl (40 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 40.8/40.8 kB 159.5 MB/s eta 0:00:00
Collecting sphinxcontrib-devhelp
  Downloading sphinxcontrib_devhelp-1.0.2-py2.py3-none-any.whl (84 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 84.7/84.7 kB 181.3 MB/s eta 0:00:00
Collecting sphinxcontrib-jsmath
  Downloading sphinxcontrib_jsmath-1.0.1-py2.py3-none-any.whl (5.1 kB)
Collecting Pygments>=2.12
  Downloading Pygments-2.13.0-py3-none-any.whl (1.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.1/1.1 MB 216.8 MB/s eta 0:00:00
Collecting sphinxcontrib-applehelp
  Downloading sphinxcontrib_applehelp-1.0.2-py2.py3-none-any.whl (121 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 121.2/121.2 kB 197.0 MB/s eta 0:00:00
Collecting sphinxcontrib-qthelp
  Downloading sphinxcontrib_qthelp-1.0.3-py2.py3-none-any.whl (90 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 90.6/90.6 kB 190.9 MB/s eta 0:00:00
Collecting sphinxcontrib-serializinghtml>=1.1.5
  Downloading sphinxcontrib_serializinghtml-1.1.5-py2.py3-none-any.whl (94 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 94.0/94.0 kB 186.9 MB/s eta 0:00:00
Collecting pytz>=2015.7
  Downloading pytz-2022.4-py2.py3-none-any.whl (500 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 500.8/500.8 kB 211.6 MB/s eta 0:00:00
Collecting typing-extensions>=3.6.4
  Downloading typing_extensions-4.4.0-py3-none-any.whl (26 kB)
Collecting zipp>=0.5
  Downloading zipp-3.9.0-py3-none-any.whl (5.8 kB)
Collecting pyparsing!=3.0.5,>=2.0.2
  Downloading pyparsing-3.0.9-py3-none-any.whl (98 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 98.3/98.3 kB 186.9 MB/s eta 0:00:00
Collecting charset-normalizer<3,>=2
  Downloading charset_normalizer-2.1.1-py3-none-any.whl (39 kB)
Collecting certifi>=2017.4.17
  Downloading certifi-2022.9.24-py3-none-any.whl (161 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 161.1/161.1 kB 199.7 MB/s eta 0:00:00
Collecting idna<4,>=2.5
  Downloading idna-3.4-py3-none-any.whl (61 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 61.5/61.5 kB 171.0 MB/s eta 0:00:00
Collecting urllib3<1.27,>=1.21.1
  Downloading urllib3-1.26.12-py2.py3-none-any.whl (140 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 140.4/140.4 kB 183.3 MB/s eta 0:00:00
Building wheels for collected packages: mock, livereload
  Building wheel for mock (setup.py): started
  Building wheel for mock (setup.py): finished with status 'done'
  Created wheel for mock: filename=mock-1.0.1-py3-none-any.whl size=23772 sha256=399ae7c23e006dbb58834ff84ef8acc665fe394365ef82b265575a4f177407ec
  Stored in directory: /tmp/pip-ephem-wheel-cache-3zq2mvt7/wheels/7a/94/b1/0fdc5339a5bd487a5cc15421ec200d9ba3e2aa7190e4a727f1
  Building wheel for livereload (setup.py): started
  Building wheel for livereload (setup.py): finished with status 'done'
  Created wheel for livereload: filename=livereload-2.6.3-py2.py3-none-any.whl size=24725 sha256=e0fc0722a7dbaa1c68730d49f80a9de2918f6af8d7d60b2a6daaced555ec6ae9
  Stored in directory: /tmp/pip-ephem-wheel-cache-3zq2mvt7/wheels/d4/f2/03/55f37093eb8cb0c89d7efb206f792dba55cd5bd67b1c5b1ce1
Successfully built mock livereload
Installing collected packages: snowballstemmer, pytz, mock, commonmark, alabaster, zipp, urllib3, typing-extensions, tornado, sphinxcontrib-serializinghtml, sphinxcontrib-qthelp, sphinxcontrib-jsmath, sphinxcontrib-htmlhelp, sphinxcontrib-devhelp, sphinxcontrib-applehelp, six, PyYAML, pyparsing, Pygments, pillow, MarkupSafe, imagesize, idna, docutils, charset-normalizer, certifi, babel, requests, packaging, livereload, Jinja2, importlib-metadata, sphinx, Markdown, click, recommonmark, mkdocs
Successfully installed Jinja2-3.1.2 Markdown-3.4.1 MarkupSafe-2.1.1 PyYAML-6.0 Pygments-2.13.0 alabaster-0.7.12 babel-2.10.3 certifi-2022.9.24 charset-normalizer-2.1.1 click-8.1.3 commonmark-0.9.1 docutils-0.19 idna-3.4 imagesize-1.4.1 importlib-metadata-5.0.0 livereload-2.6.3 mkdocs-1.0.4 mock-1.0.1 packaging-21.3 pillow-9.2.0 pyparsing-3.0.9 pytz-2022.4 recommonmark-0.5.0 requests-2.28.1 six-1.16.0 snowballstemmer-2.2.0 sphinx-5.2.3 sphinxcontrib-applehelp-1.0.2 sphinxcontrib-devhelp-1.0.2 sphinxcontrib-htmlhelp-2.0.0 sphinxcontrib-jsmath-1.0.1 sphinxcontrib-qthelp-1.0.3 sphinxcontrib-serializinghtml-1.1.5 tornado-6.2 typing-extensions-4.4.0 urllib3-1.26.12 zipp-3.9.0

[rtd-command-info] start-time: 2022-10-11T16:55:45.475987Z, end-time: 2022-10-11T16:55:47.405673Z, duration: 1, exit-code: 0
/home/docs/checkouts/readthedocs.org/user_builds/andy-pycontrol-docs/envs/v1.6/bin/python -m pip install --exists-action=w --no-cache-dir -r requirements.txt
Collecting mkdocs==1.3.0
  Downloading mkdocs-1.3.0-py3-none-any.whl (3.7 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.7/3.7 MB 48.4 MB/s eta 0:00:00
Requirement already satisfied: jinja2==3.1.2 in /home/docs/checkouts/readthedocs.org/user_builds/andy-pycontrol-docs/envs/v1.6/lib/python3.7/site-packages (from -r requirements.txt (line 2)) (3.1.2)
Requirement already satisfied: packaging>=20.5 in /home/docs/checkouts/readthedocs.org/user_builds/andy-pycontrol-docs/envs/v1.6/lib/python3.7/site-packages (from mkdocs==1.3.0->-r requirements.txt (line 1)) (21.3)
Requirement already satisfied: Markdown>=3.2.1 in /home/docs/checkouts/readthedocs.org/user_builds/andy-pycontrol-docs/envs/v1.6/lib/python3.7/site-packages (from mkdocs==1.3.0->-r requirements.txt (line 1)) (3.4.1)
Requirement already satisfied: importlib-metadata>=4.3 in /home/docs/checkouts/readthedocs.org/user_builds/andy-pycontrol-docs/envs/v1.6/lib/python3.7/site-packages (from mkdocs==1.3.0->-r requirements.txt (line 1)) (5.0.0)
Collecting watchdog>=2.0
  Downloading watchdog-2.1.9-py3-none-manylinux2014_x86_64.whl (78 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 78.4/78.4 kB 186.0 MB/s eta 0:00:00
Requirement already satisfied: PyYAML>=3.10 in /home/docs/checkouts/readthedocs.org/user_builds/andy-pycontrol-docs/envs/v1.6/lib/python3.7/site-packages (from mkdocs==1.3.0->-r requirements.txt (line 1)) (6.0)
Requirement already satisfied: click>=3.3 in /home/docs/checkouts/readthedocs.org/user_builds/andy-pycontrol-docs/envs/v1.6/lib/python3.7/site-packages (from mkdocs==1.3.0->-r requirements.txt (line 1)) (8.1.3)
Collecting mergedeep>=1.3.4
  Downloading mergedeep-1.3.4-py3-none-any.whl (6.4 kB)
Collecting pyyaml-env-tag>=0.1
  Downloading pyyaml_env_tag-0.1-py3-none-any.whl (3.9 kB)
Collecting ghp-import>=1.0
  Downloading ghp_import-2.1.0-py3-none-any.whl (11 kB)
Requirement already satisfied: MarkupSafe>=2.0 in /home/docs/checkouts/readthedocs.org/user_builds/andy-pycontrol-docs/envs/v1.6/lib/python3.7/site-packages (from jinja2==3.1.2->-r requirements.txt (line 2)) (2.1.1)
Collecting python-dateutil>=2.8.1
  Downloading python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 247.7/247.7 kB 214.2 MB/s eta 0:00:00
Requirement already satisfied: typing-extensions>=3.6.4 in /home/docs/checkouts/readthedocs.org/user_builds/andy-pycontrol-docs/envs/v1.6/lib/python3.7/site-packages (from importlib-metadata>=4.3->mkdocs==1.3.0->-r requirements.txt (line 1)) (4.4.0)
Requirement already satisfied: zipp>=0.5 in /home/docs/checkouts/readthedocs.org/user_builds/andy-pycontrol-docs/envs/v1.6/lib/python3.7/site-packages (from importlib-metadata>=4.3->mkdocs==1.3.0->-r requirements.txt (line 1)) (3.9.0)
Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in /home/docs/checkouts/readthedocs.org/user_builds/andy-pycontrol-docs/envs/v1.6/lib/python3.7/site-packages (from packaging>=20.5->mkdocs==1.3.0->-r requirements.txt (line 1)) (3.0.9)
Requirement already satisfied: six>=1.5 in /home/docs/checkouts/readthedocs.org/user_builds/andy-pycontrol-docs/envs/v1.6/lib/python3.7/site-packages (from python-dateutil>=2.8.1->ghp-import>=1.0->mkdocs==1.3.0->-r requirements.txt (line 1)) (1.16.0)
Installing collected packages: watchdog, pyyaml-env-tag, python-dateutil, mergedeep, ghp-import, mkdocs
  Attempting uninstall: mkdocs
    Found existing installation: mkdocs 1.0.4
    Uninstalling mkdocs-1.0.4:
      Successfully uninstalled mkdocs-1.0.4
Successfully installed ghp-import-2.1.0 mergedeep-1.3.4 mkdocs-1.3.0 python-dateutil-2.8.2 pyyaml-env-tag-0.1 watchdog-2.1.9

[rtd-command-info] start-time: 2022-10-11T16:55:48.004155Z, end-time: 2022-10-11T16:55:48.066974Z, duration: 0, exit-code: 0
cat mkdocs.yml
copyright: Copyright &copy; 2016 - 2022 Thomas Akam
docs_dir: docs
extra_css:
- /_/static/css/badge_only.css
- /_/static/css/readthedocs-doc-embed.css
extra_javascript:
- readthedocs-data.js
- /_/static/core/js/readthedocs-doc-embed.js
- /_/static/javascript/readthedocs-analytics.js
google_analytics: null
nav:
- Home: index.md
- User Guide:
  - Graphical user interface (GUI): user-guide/graphical-user-interface.md
  - Programming tasks: user-guide/programming-tasks.md
  - Hardware: user-guide/hardware.md
  - Synchronisation: user-guide/synchronisation.md
  - pyControl data: user-guide/pycontrol-data.md
  - Troubleshooting: user-guide/troubleshooting.md
- Contributing: contributing.md
- About:
  - Release Notes: about/release-notes.md
  - License: about/license.md
repo_url: https://github.com/pycontrol/pycontrol-docs
site_author: Thomas Akam
site_description: Open source, Python based, behavioural experiment control.
site_name: pyControl
site_url: https://pycontrol.readthedocs.io/
theme:
  name: readthedocs

[rtd-command-info] start-time: 2022-10-11T16:55:48.480622Z, end-time: 2022-10-11T16:55:49.083638Z, duration: 0, exit-code: 0
/home/docs/checkouts/readthedocs.org/user_builds/andy-pycontrol-docs/envs/v1.6/bin/python -m mkdocs build --clean --site-dir _build/html --config-file mkdocs.yml
INFO     -  Cleaning site directory
INFO     -  Building documentation to directory: /home/docs/checkouts/readthedocs.org/user_builds/andy-pycontrol-docs/checkouts/v1.6/_build/html
INFO     -  Documentation built in 0.30 seconds
```

**Comment by ThomasAkam** [comment]
Thanks Andy.  I think I worked out what was causing the problem.  I originally created a tag on the main branch for the v1.6 version but this did not build properly, so I deleted the tag and created a new v1.6 branch.  However, it seems the v1.6 version on readthedocs created from the original tag did not get deleted when I deleted the tag, possible because it was marked as active at the time (see [here](https://github.com/readthedocs/readthedocs.org/issues/5752)) .  Instead a new version was created on readthedocs called v1.6_a pointing to the branch not the now deleted tag, so when I tried to build v1.6 it was still using the commit the tag had pointed to.  I fixed it by creating a new v1.6 tag pointing to the correct commit.
